pkgs/applications/editors/rstudio/default.nix

```diff
@@ -27,7 +27,7 @@
 , yaml-cpp
 , soci
 , postgresql
-, nodejs
+, nodejs_20
```

Comment:
@jbedo what do you think? At Posit they use 20.x to build. Do you think it's more prudent to use the same major version, or it doesn't matter?

Reply:
If 20 is available we may as well use it and try to match the upstream build process to minimise unexpected behaviours.